server/server_test.go

```diff
@@ -159,6 +159,42 @@ func TestGetKnownHosts(t *testing.T) {
 	}
 }
 
+func TestStatus(t *testing.T) {
+	c, err := generateContext()
+	if err != nil {
+		t.Fatalf("Error generating context: %s", err)
+	}
+	defer c.db.Close()
+
+	req, err := generateRequest()
+	if err != nil {
+		t.Fatalf("Error generating context: %s", err)
+	}
+
+	rec := httptest.NewRecorder()
+	c.Status(rec, req)
+
+	rec.Flush()
+	if rec.Code != 200 {
+		t.Fatalf("Request to /_status failed with %d", rec.Code)
+	}
+
+	body, _ := ioutil.ReadAll(rec.Body)
+	expected, err := ioutil.ReadFile("testdata/status_ok.json")
```

Comment:
could we just put this as a byte string here?  Seems silly given how short it is

Reply:
yeah, that's fair!